services/mov/mover.go

```diff
@@ -0,0 +1,21 @@
+// Package handler is responsible for logistics
```

Comment:
Package mov or handler and what is "responsible for logistics"? What is responsability of this package?